common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2111,8 +2111,7 @@ public static enum ConfVars {
         "org.apache.hadoop.hive.serde2.MetadataTypedColumnsetSerDe," +
         "org.apache.hadoop.hive.serde2.columnar.LazyBinaryColumnarSerDe," +
         "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe," +
-        "org.apache.hadoop.hive.serde2.lazybinary.LazyBinarySerDe," +
-        "org.apache.hadoop.hive.serde2.OpenCSVSerde",
+        "org.apache.hadoop.hive.serde2.lazybinary.LazyBinarySerDe",
```

Comment:
I want to confirm why we can remove this one. What should users do if they want to use CSV? Or what if the existing Hive tables are created with `OpenCSVSerde`?